.github/workflows/ci.yml

```diff
@@ -306,10 +317,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version:
-          - "7.2"
-          - "7.3"
-          - "7.4"
+        php-version: ${{ fromJson(needs.supported-versions-matrix.outputs.version) }}
```

Comment:
@sebastianbergmann 

Looks like we will be running additional jobs here, for

- "8.0
- "8.1"
- "8.2"